packages/instant/public/index.html

```diff
@@ -34,33 +34,48 @@
             EXPONENTIAL_AT: 1000,
             DECIMAL_PLACES: 78,
         });
-        const providedOrder = {
-            senderAddress: '0x0000000000000000000000000000000000000000',
-            makerAddress: '0x14e2f1f157e7dd4057d02817436d628a37120fd1',
-            takerAddress: '0x0000000000000000000000000000000000000000',
-            makerFee: new BigNumber('0'),
-            takerFee: new BigNumber('0'),
-            makerAssetAmount: new BigNumber('100000000000000000000'),
-            takerAssetAmount: new BigNumber('10000000000000000'),
-            makerAssetData: '0xf47261b00000000000000000000000002002d3812f58e35f0ea1ffbf80a75a38c32175fa',
-            takerAssetData: '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
-            expirationTimeSeconds: new BigNumber('1591858800'),
-            feeRecipientAddress: '0x0000000000000000000000000000000000000000',
-            salt: new BigNumber(
-                '54983920541892966634674340965984367456810207583416050222519063020710969340046',
-            ),
-            signature:
-                '0x1b949656218421c845995457303569a656764afa2b979d41dcefff0009d57ce15001490268bc7caa4269894fd83b741465fc5a7a53eda6ece17eb91fb32655d83703',
-            exchangeAddress: '0x35dd2932454449b14cee11a94d3674a936d5d7b2',
-        };
+        const providedOrders = [
+            {
+                "senderAddress": "0x0000000000000000000000000000000000000000",
```

Comment:
Why did we move to double quotes here? we should be consistent with the rest of the file